README.md

```diff
@@ -14,100 +14,141 @@ but easier to bootstrap and has own name. That's why it easier to have multiple
-    * [Terraform](https://www.terraform.io/intro/getting-started/install.html)
+Configure your credentials in `.env` file.
+Use [.env.example](https://github.com/dashpay/dash-network-deploy/blob/master/examples/.env.example) as example.
 
-2. Install pre-requisite Ansible roles
+### Networks definition
 
-    ```bash
-    ansible-galaxy install -r ansible/requirements.yml
-    ```
+Please find examples of network configurations
+in [examples directory](https://github.com/dashpay/dash-network-deploy/tree/master/examples/).
 
-3. Ensure Python netaddr package installed locally
+Name of files are equal to Dash network names.
 
-    ```bash
-    pip install -U netaddr
-    ```
-    
-    * Note: You may need to run the above command with "pip2" instead of "pip" if
-      your default Python installation is version 3 (e.g. OSX + Homebrew).
+Terraform configuration defined in `*.tfvars` files.
+All available options you will find
+in [variables.tf](https://github.com/dashpay/dash-network-deploy/blob/master/terraform/aws/variables.tf) file.
 
-## Configuration
+Ansible configuration in `*.yaml` files.
+[group_vars/all](https://github.com/dashpay/dash-network-deploy/blob/master/ansible/group_vars/all)
+file contains the majority of playbook options.
+The rest of them defined in [ansible roles](https://github.com/dashpay/dash-network-deploy/tree/master/ansible/roles).
 
-Configure your credentials in `.env` file:
+### Using git
 
-```bash
-cp .env.example .env
-$EDITOR .env
+Please don't forget to include in your `.gitignore`:
+```
+.env
+*.inventory
+*.ovpn
 ```
 
 ### Evolution configuration
+
 Several Ansible's arguments should be specified in `.env` file to deploy Evolution's services:
+
 ```bash
 ANSIBLE_ARGUMENTS="-e evo_services=true \
                    -e insight_image=<path-to-image> \
                    -e drive_image=<path-to-image> \
                    -e dapi_image=<path-to-image> \
                    -e dashd_image=<path-to-image>"
 ```
-For Docker images stored in AWS ECR the argument `-e aws_ecr_login=true` is required.
 
-## Networks definition
+For Docker images stored in AWS ECR, the argument `-e aws_ecr_login=true` is required.
 
-Use available or create your own network configuration in [networks directory](networks).
-Name of files in [networks directory](networks) are equal to Dash network names.
+## Deployment
 
-Terraform configuration defined in `*.tfvars` files.
-All available options you will find in [variables.tf](terraform/aws/variables.tf) file.
-Ansible configuration in `*.yaml` files.
-[group_vars/all](ansible/group_vars/all) file contains the majority of playbook options.
-The rest of them you will find in `roles/*/vars/main.yml` files.
-
-## Network deployment
-
-To deploy Dash Network run `bin/deploy` command with particular network name:
+To deploy Dash Network use `deploy` command with particular network name:
 
 ```bash
-./bin/deploy <network_name>
+docker run -ti --rm \
+           -v "$PWD:/networks" \
```

Comment:
Small fix: `-v "$PWD/networks:/networks"` (Same for destroy and test)

Reply:
The idea is that with Docker you don't need to clone the repository. Network definitions and `.env` should be in your workdir which you mount to the container.

Reply:
Oh, I see the idea 👍